config/locales/curation_concerns.en.yml

```diff
@@ -159,6 +159,8 @@ en:
   file_manager:
     link_text: 'File Manager'
   simple_form:
+    required:
+      html: '<span class="label label-info required-tag">required</span>'
```

Comment:
After looking into this a bit more simple_form has the attribute element built in. Since we're no longer using the "*" it's use wouldn't make semantic sense. https://github.com/plataformatec/simple_form/blob/ac63da0f0b38df82385493f25f649e320b0acb4a/lib/simple_form/components/labels.rb#L10

The documented/simple_form way to use custom required markup is to define html in the yaml file. https://github.com/plataformatec/simple_form/blob/ac63da0f0b38df82385493f25f649e320b0acb4a/lib/generators/simple_form/templates/config/locales/simple_form.en.yml#L8-L10

Considering the trade-offs I think this implementation is the "right way" even though the defined yaml html will stay the same with different languages. We can also avoid writing custom code this way.